src/View/Contact/contact.html.twig

```diff
@@ -0,0 +1,34 @@
+  
+
+{% block contact %}
+<div class="form">
+    <div class="contact_overlay">
+    
+        <h1 class="txt_form">UN AVIS, UNE REMARQUE, NOUS SOMMES PRENEUR DE TOUT BON CONSEIL
```

Comment:
please put a more undurstandable classNAme, since it is more a title than a text